docs/LUA.md

```diff
@@ -1,23 +1,33 @@
-# mLRS Documentation: mLRS Configuration Lua Script #
+# mLRS Documentation: mLRS Lua Script #
 
 ([back to main page](../README.md))
 
-The mLRS configuration Lua script provides the most convenient approach to set the Tx and Rx module's configuration settings.
+The mLRS Lua script provides the most convenient way to change the Tx and Rx module's settings.
 
-The Lua script works on both OpenTx and EdgeTx radios.  There are two different versions of the Lua script which depends on your radio's display:
+The Lua script works on both OpenTx and EdgeTx radios but there are two different versions depending on the display of your radio:
 1. If your radio has a 480x272 color screen (e.g. Jumper T16, Radiomaster TX16S) then use the "mLRS.lua" file
 2. If your radio has a black and white screen (e.g. Frsky Taranis X9E, Radiomaster Zorro) then use the "mLRS-bw.lua" file
 
-Note: The Lua script for radios with a black and white screen is limited in functionality and only allows for the BindPhrase, Mode, TxPwr, RxPwr, and RxOutMode parameters to be set.
+Depending on the device, some parameters are not available for configuration or cannot be changed. For instance, for a device which doesn't support a buzzer the parameter "Buzzer" is not available, and for a device which doesn't support diversity the parameter "Diversity" cannot be changed. Parameters which are not available are not displayed on the screen (i.e. the list of shown parameters can vary depending on the device), and parameters which cannot be changed are displayed with the current selection but are greyed out and cannot be edited.
```

Comment:
this I find a little bit confusing since it reads as if it would hold for both scripts, but the text as such only applies to the color lua. Not sure with what words to get around this easily.

Reply:
maybe an approach would be to insert a chapter "Color Lua Script" an move that kind of text in there.